frontend/app/.server/routes/helpers/protected-renew-route-helpers.ts

```diff
@@ -196,24 +200,31 @@ export function clearProtectedRenewState({ params, session }: ClearStateArgs) {
 interface StartArgs {
   id: string;
   session: Session;
+  appContainer: AppContainerProvider;
 }
 
 /**
  * Starts protected renew state.
  * @param args - The arguments.
  * @returns The initial protected renew state.
  */
-export function startProtectedRenewState({ id, session }: StartArgs) {
+export async function startProtectedRenewState({ id, session, appContainer }: StartArgs) {
   const log = getLogger('protected-renew-route-helpers.server/startProtectedRenewState');
   const parsedId = idSchema.parse(id);
+  const sessionName = getSessionName(parsedId);
+
+  const clientApplicationService = appContainer.get(TYPES.domain.services.ClientApplicationService);
+  const clientApplication = await clientApplicationService.findClientApplicationBySin(session.get('userInfoToken').sin);
+  invariant(clientApplication, 'Expected client application to be defined');
 
+  // TODO: create a mapper function from clientApplication to initialState?
   const initialState: ProtectedRenewState = {
     id: parsedId,
     editMode: false,
-    children: [],
+    applicantInformation: clientApplication.applicantInformation,
+    children: clientApplication.children.map((child) => ({ id: randomUUID(), ...child, ...child.information })),
   };
```

Comment:
My thinking is that we store the immutable `clientApplication` as its own object in state and any further modifications will be kept as a separate object(s). 

The state mapper will take the unmodified `clientApplication` and any new data to build the new renewal DTO, similar to how it's done in the unauthenticated area.

The member selection can reference this `clientApplication` object in state, whereas the review information can originally reference this object and render the new data in state depending on if the user updates their info.

Reply:
hmm, okay.  I'll have to go back to the drawing board.  I understand you though.

Reply:
alright, so I tried some different permutations, but came to the same conclusion.  We have fields on both `applicantInformation` and `children` (`externallyReviewed`, `previouslyReviewed`, `firstName`, `lastName`, `id` (for the child that's part of the path param)) that are coupled to the member-selection screen. Before we land on this screen, we need values populated in state.  Ultimately I think we're going to need a mapper, but that's going to be a pretty big effort for this PR.  My intention is to for this mapper to be called in the `startProtectedRenewState` as in this PR.  I'm not sure where `externallyReviewed`, `previouslyReviewed` are coming from, or if those are native to our application/state.  Either way, this is more complicated than it initially appears.

Reply:
For the primary applicant that's doable, but for the children I don't think there's an easy work around, unless I'm missing something obvious.